App/SB/components/CommentCard/statics/styles.js

```diff
@@ -12,7 +12,8 @@ export default StyleSheet.create({
   },
   content: {
     flexDirection: 'row',
-    margin: 12
+    width: '100%',
+    padding: 12
```

Comment:
Using `padding` here because using `margin` was behaving strangely. It would put the correct margin on the top, left and bottom, but no margin on the right side. Ever seen that before @andrewxhill ?